### DISCUSSION TITLE
Deployment problem

### DISCUSSION THREAD

**Discussion by halwesit** [keystonejs/keystone#7933] [Questions]
Hello Everyone

I have created a KystoneJS app, but I have a problem with deployment, first I tried shared hosting and I couldn't find an entry point for the application start, second, I tried to make a docker image, but it was too large, but the image was about 4GB.

I don't want to deploy it into Heroku, Azure or railway as mentioned in the docs.

**Comment by flexdinesh** [comment]
What do you mean by _shared hosting_? To start Keystone server, you should have run `yarn build` before and run `yarn start`. The command looks for a file called `keystone.ts` in the current working directory.
